tst/com/amazon/corretto/crypto/provider/test/AesCtrDrbgTest.java

```diff
@@ -198,7 +198,7 @@ public void nistTestVectors() throws ReflectiveOperationException, SecurityExcep
         int tests = 0;
 
         try (final Scanner in = new Scanner(
-                AmazonCorrettoCryptoProvider.class.getResourceAsStream("/test-data/ctr-drbg.txt"), StandardCharsets.US_ASCII.name())) {
+                AmazonCorrettoCryptoProvider.class.getResourceAsStream("testdata/ctr-drbg.txt"), StandardCharsets.US_ASCII.name())) {
```

Comment:
As discussed, this may have problems when testing older versions of this library (which we must support).  Try the following code instead

```
Class<?> loaderClass =Class.forName("com.amazon.corretto.crypto.provider.Loader"); // Maybe do this once and make it a public final constant on `TestUtil`?
// Misc code
new Scanner((InputStream) TestUtil.sneakyInvoke(loaderClass, "getTestData", "ctr-drbg.txt"), StandardCharsets.US_ASCII.name());
```